api/external/infra_proxy/request/orgs.proto

```diff
@@ -4,37 +4,53 @@ package chef.automate.api.infra_proxy.request;
 option go_package = "github.com/chef/automate/api/external/infra_proxy/request";
 
 message CreateOrg {
+    // Name of the Org.
```

Comment:
Probably should check with UX or product, but to me it seems better not to abbreviate to "Org". Also, should not be capitalized.
```suggestion
    // Name of the organization.
```
Note that this change should be applied across the comments in all the protos herein.

Reply:
I think all of these should probably say Chef Organization